apps/chat/src/components/Chat/Publish/ReviewApplicationDialogView.tsx

```diff
@@ -0,0 +1,128 @@
+import { IconBulb } from '@tabler/icons-react';
+
+import { useTranslation } from 'next-i18next';
+
+import { getFolderIdFromEntityId } from '@/src/utils/app/folders';
+import { ApiUtils } from '@/src/utils/server/api';
+
+import { Translation } from '@/src/types/translation';
+
+import { ApplicationSelectors } from '@/src/store/application/application.reducers';
+import { useAppSelector } from '@/src/store/hooks';
+
+import { PublicationControls } from './PublicationChatControls';
+
+export function ReviewApplicationDialogView() {
+  const { t } = useTranslation(Translation.Chat);
+  const application = useAppSelector(
+    ApplicationSelectors.selectApplicationDetail,
+  );
+  const entity = application && {
```

Comment:
Meybe we can create `ReviewApplicationDialog` and `ReviewApplicationDialogView` and render `ReviewApplicationDialogView` only if `entity !== undefined`